src/zenml/integrations/tensorflow/materializers/keras_materializer.py

```diff
@@ -13,10 +13,13 @@
 #  permissions and limitations under the License.
 """Implementation of the TensorFlow Keras materializer."""
 
+import os
 import tempfile
 from typing import TYPE_CHECKING, Any, ClassVar, Dict, Tuple, Type
 
-from tensorflow import keras
+import keras
```

Comment:
I have tested this myself by installing zenml along with the `tensorflow` integration, followed by a direct `import keras` and it worked. Just in case tagging @avishniakov here to resolve the question.

Reply:
I reworked this to be on the safe side.